cub/cub/thread/thread_scan.cuh

```diff
@@ -164,6 +164,85 @@ ThreadScanExclusive(T (&input)[LENGTH], T (&output)[LENGTH], ScanOp scan_op, T p
   return ThreadScanExclusive<LENGTH>((T*) input, (T*) output, scan_op, prefix, apply_prefix);
 }
 
+/**
```

Comment:
suggestion: Should we document that output items beyond `valid_items` are not being written to?

Reply:
Same applies for inclusive.